### PR TITLE
tikv: update CDC config

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1260,6 +1260,30 @@ raftdb 相关配置项。
...
 
+## cdc
+
+用于 TiCDC 变更数据捕捉相关的配置项。
+
+### `min-ts-interval`
+
++ 定期推进 resolved ts 的时间间隔。
++ 默认值：1s
+
```
Comment (overvenus) [inline tikv-configuration-file.md:1271]:
```suggestion
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1260,6 +1260,25 @@ raftdb 相关配置项。
 + 默认值：CPU * 0.75，但最大为 32
 + 最小值：1
 
+## cdc
+
+用于 TiCDC 变更数据捕捉相关的配置项。
+
+### `min-ts-interval`
+
++ 定期推进 resolved ts 的时间间隔。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1269]:
```suggestion
+ 定期推进 Resolved TS 的时间间隔。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1260,6 +1260,25 @@ raftdb 相关配置项。
...
++ 默认值：1s
+
+### `old-value-cache-size` <span class="version-mark">从 v5.0 版本开始引入</span>
+
++ 缓存在内存中的 TiCDC Old Value 的条目个数。
++ 默认值：1024
+
+### `incremental-scan-speed-limit` <span class="version-mark">从 v5.0 版本开始引入</span>
+
++ 增量扫历史数据速度的上限。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1279]:
```suggestion
+ 增量扫描历史数据的速度上限。
```

**Review by TomShawn — COMMENTED** [review]
Global comment: 如有取值范围，请添加取值范围

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1260,6 +1260,25 @@ raftdb 相关配置项。
...
++ 默认值：128MB/s
```
Comment (amyangfei) [inline tikv-configuration-file.md:1280]: 可以加一下配置的单位，是数据量大小，譬如 `KB|MB|GB`?

Comment (overvenus) [reply tikv-configuration-file.md:1280]:
```suggestion
+ 默认值：128MB，即 128MB 每秒。
```

**Review by amyangfei — APPROVED** [review]

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1260,6 +1260,25 @@ raftdb 相关配置项。
 + 默认值：CPU * 0.75，但最大为 32
 + 最小值：1
 
+## cdc <span class="version-mark">从 v4.0.5 版本开始引入</span>
+
+用于 TiCDC 变更数据捕捉相关的配置项。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1265]:
```suggestion
用于 TiCDC 捕捉变更数据相关的配置项。
```

**Comment by TomShawn** [comment]
Follow-up: remove the version marks.

**Pull request merged**